<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICPasswordModifyDialog</class>
 <widget class="QDialog" name="ICPasswordModifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Old Super Password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="oldPassword"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>New Super Password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="newPassword"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Confirm New Super Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="newPassword_2"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
